dora/build.rs

```diff
@@ -50,7 +51,10 @@ fn main() -> Result<(), Box<dyn Error>> {
             None => continue,
         }
 
-        copy(root_path.join(f.path()), out_path.join(f.path()))?;
+        if f.file_type().is_file() {
```

Comment:
Shouldn't this be always true here?

Reply:
Could also be a symlink.

Reply:
Hmm, but we already checked is_file() a few lines above, right? So this should always be true?